cmake-modules/FolderList.cmake

```diff
@@ -2,34 +2,40 @@ macro(GetFolderList project)
     message ("project found ${project}")
     message ("FLAG VALUE : ${FETCH_SOURCE_DEPS}")
     if(${project} STREQUAL CERTIFICATES)
-        DownloadDepVersion(sdk/core azure-core 1.5.0)
-        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+        DownloadDepVersion(sdk/core azure-core)
+        DownloadDepVersion(sdk/identity azure-identity)
```

Comment:
This change of removing the version strings has broken the CI:
https://github.com/Azure/azure-sdk-for-cpp/issues/4810